libraries/AP_DDS/README.md

```diff
@@ -129,7 +130,7 @@ Follow the instructions for the following:
 * Do "Creating the micro-ROS agent"
 * Source your ROS workspace
 
-- Run microROS agent with the following command
+Run microROS agent with the following command to test the agent
```

Comment:
Remove `to test the agent`. It's not just used for testing, you need to do this every time you want to use DDS, at least in the current system.